### ISSUE TITLE
Node.js v4.0.0 Release Checklist for nodejs.org

### ISSUE THREAD

**Issue by rvagg** [nodejs/new.nodejs.org#101]
See https://github.com/nodejs/node/issues/2522 for ongoing details; the current plan is for it to be released in a bit over 24 hours from now so I want to make sure that (a) we have people available who can fix up the website if needed and (b) we capture all of the items that we know need doing on the website end.

The obvious differences between v4.x and v0.12.x for the website that I can think of are:
 * New download file types and adjusted old ones—armv6, armv7, arm64, no "universal" .pkg for OSX, no 32-bit OSX, .xz versions of all of the tarballs.
 * New style CHANGELOG meaning we can do more than just list commits.

What do we need to get done?

 - [x] New download page, it looks like @phillipj has this ready to go here: https://github.com/nodejs/new.nodejs.org/pull/87
 - [ ] Release blog post, we're not auto-generating these from core any more, it'll have to be done manually for now I suppose. Current release proposal with WIP changelog entry: <https://github.com/nodejs/node/pull/2715>, also should have some additional blog-post style text to go along with it with a quick summary of what v4 is all about and pointing to any resources we have for people to read. I can work on the text here and PR to this repo for feedback.

Is there anything else I'm not seeing? I'm assuming that nodejs.org/dist/index.json will be automatically picked up and the downloads pages generated for them, is that correct?

**Comment by fhemberger** [comment]
- [ ] Run `scripts/load-versions.js` after release once to get the full versions list (needed to determine current version and release history).
- [ ] Remove the comment here: https://github.com/nodejs/new.nodejs.org/blob/master/layouts/download-releases.hbs#L13 (and remove the class from h2, we're not using this style any longer).

**Comment by sambthompson** [comment]
- [ ] Repoint Changelog on front page from https://github.com/nodejs/node-v0.x-archive/blob/master/ChangeLog to https://github.com/nodejs/node/blob/master/CHANGELOG.md, reversing fix in #84

**Comment by fhemberger** [comment]
I've created a separate [node-4.0-release](https://github.com/nodejs/new.nodejs.org/tree/node-4.0-release) branch. Please commit required changes there, so we just need to merge it once the release is finished.

**Comment by phillipj** [comment]
The `release-post.js` script found in #88 could be worth testing, atleast for filling in the changelog, files and shasums. It'll create `locale/en/blog/release/v4.0.0.md` locally and can be edited by hand from there..

It fetches data from three different sources, which all have to be updated / merged into node's repo master before running the script:
- https://nodejs.org/dist/index.json
- https://raw.githubusercontent.com/nodejs/node/master/CHANGELOG.md
- https://nodejs.org/dist/v4.0.0/SHASUMS256.txt.asc

**Comment by fhemberger** [comment]
@phillipj :+1: Let's do this.

**Comment by rvagg** [comment]
So I can edit the resulting file and add some custom text about the release after it's been created? And `release-post.js` is run by hand to do the generating, not on the server at deploy time?

**Comment by phillipj** [comment]
> So I can edit the resulting file and add some custom text about the release after it's been created?

Yes, followed by you commiting it to the repo.

It's all manual by hand atm. We could ofc discuss something fancier / automatic later.

**Comment by rvagg** [comment]
- [ ] Update the ES6 documentation @ https://nodejs.org/en/docs/es6/, note the talk of arrow functions is no longer correct for v4. We need a volunteer to update this.

**Comment by rvagg** [comment]
all done I think

**Comment by fhemberger** [comment]
@rvagg https://github.com/nodejs/node/blob/master/CHANGELOG.md still shows v3.3.0 as latest version.

**Comment by rvagg** [comment]
fixed now with a merge of https://github.com/nodejs/node/pull/2742, I probably could have got away with not waiting for review but pehaps we ought to use tagged links to CHANGELOG.md in future, e.g. https://github.com/nodejs/node/blob/v4.0.0/CHANGELOG.md

**Comment by fhemberger** [comment]
Yes, we have already switched the link on the home page to the tagged one.
